Fix DV360 docs link (#6174)
segmentio/segment-docs: src/connections/destinations/catalog/actions-display-video-360/index.md

Google's [Display & Video (DV360)](https://marketingplatform.google.com/about/display-video-360/){:target="_blank"} service is an end-to-end campaign management tool that enables enterprise customers to plan, measure, and run display and video advertisements. 

> info ""

> > You can connect to a Google Ad Manager account. For more information, see [4. Create an audience and finish DV360 configuration](#create-an-audience-and-finish-dv360-configuration) below. Set **User-Role Granted** to `Publisher` if you plan to connect to Google Ad Manager.

Reviewer: ```suggestion
> You can connect to a Google Ad Manager account. For more information, see [Create an audience and finish DV360 configuration](#create-an-audience-and-finish-dv360-configuration) below. Set **User-Role Granted** to `Publisher` if you plan to connect to Google Ad Manager.
```

### Enable and configure the DV360 Destination

1. From your Segment workspace, navigate to **Engage > Engage Settings > Destinations > Add Destination**, then search for **Display and Video 360 (Actions)**.

> 2. Authenticate using OAuth. We will ask for permissions to see, edit, create and delete your Audience Partner account data.

Reviewer: ```suggestion
2. Authenticate using OAuth. Segment asks for permissions to see, edit, create and delete your Audience Partner account data.
```
